src/code/modules/parent-gametes.js

```diff
@@ -0,0 +1,134 @@
+
+  organized according to the "ducks" principles described at
+  https://github.com/erikras/ducks-modular-redux with some ideas taken from
+  https://jaysoo.ca/2016/02/28/applying-code-organization-rules-to-concrete-redux-code/
+ */
+import Immutable from 'seamless-immutable';
+import { createSelector } from 'reselect';
+import { clone, cloneDeep } from 'lodash';
+import actionTypes from '../action-types';
+import { ITS_ACTORS, ITS_ACTIONS, ITS_TARGETS } from '../its-constants';
+
+/*
+ * action types
+ */
+export const GAMETES_ADDED_TO_POOL = "Gametes added to gamete pool";
+export const GAMETE_POOLS_RESET = "Gamete pools reset";
+export const GAMETE_SELECTED = "Gamete selected in pool";
+
+/*
+  state contains:
+    pools: array of two arrays, 0: father's gametes, 1: mother's gametes
+    selectedIndices: array of two indices representing the index of
+                      the selected gamete (if any) for each parent
+ */
+const initialState = Immutable({ pools: Immutable([Immutable([]), Immutable([])]),
+                                  selectedIndices: Immutable([null, null]) });
+
+/*
+ * reducer
+ */
+export default function reducer(state = initialState, action) {
+  let newState = {}, pools, selectedIndices;
+  switch (action.type) {
+    case GAMETE_POOLS_RESET:
+      return initialState;
+    case GAMETES_ADDED_TO_POOL:
+      pools = state.pools.map((pool, index) => {
+                  return index === action.sex
+                          ? pool.concat(action.gametes)
+                          : pool;
+                });
+      return state.merge({ pools });
+    case GAMETE_SELECTED:
+      selectedIndices = state.selectedIndices.map((selectedIndex, index) => {
+                                        return (index === action.sex) ? action.index : selectedIndex;
+                                      });
+      return state.merge({ selectedIndices });
+    case actionTypes.OFFSPRING_KEPT:
+      // remove selected gamete from pool after offspring drake is kept
+      newState.pools = state.pools.map((pool, poolIndex) => {
+                          return pool.map((gamete, gameteIndex) => {
+                            return gameteIndex === state.selectedIndices[poolIndex]
+                                    ? null : gamete;
+                          });
+                        });
+      // fallthrough intentional
+    case actionTypes.GAMETES_RESET:
+      newState.selectedIndices = Immutable([null, null]);
+      return state.merge(newState);
+    default:
+      return state;
+  }
+}
+
+/*
+ * selectors
+ */
+export const fatherGametePool = state => state.pools[0];
+export const motherGametePool = state => state.pools[1];
+export const gametePoolSelector = sex => sex ? motherGametePool : fatherGametePool;
+export const fatherSelectedGameteIndex = state => state.selectedIndices[0];
+export const motherSelectedGameteIndex = state => state.selectedIndices[1];
+export const fatherSelectedGamete = createSelector(
```

Comment:
They may not be used at the moment. I was thinking about what the module API should look like and it seemed reasonable that clients might want to have encapsulated access to the gametes themselves. (Plus, it gave me a chance to see how selector composition worked with the reselect library, which I thought would be a good thing to understand. 😉 )